tf2_yolov4/tools/convert_tflite.py

```diff
@@ -0,0 +1,68 @@
+"""
+Script used to create a TfLite YOLOv4 model from previously trained weights.
+"""
+
+import click
+import tensorflow as tf
+
+from tf2_yolov4.anchors import YOLOV4_ANCHORS
+from tf2_yolov4.model import YOLOv4
+
+HEIGHT, WIDTH = (640, 960)
+
+TFLITE_MODEL_PATH = "yolov4.tflite"
+
+
+def create_tflite_model(model):
+    """Converts a YOLOv4 model to a TfLite model
+
+    Args:
+        model (tensorflow.python.keras.engine.training.Model): YOLOv4 model
+
+    Returns:
+        (bytes): a binary TfLite model
+    """
+    converter = tf.lite.TFLiteConverter.from_keras_model(model)
+    converter.optimizations = [tf.lite.Optimize.DEFAULT]
+    converter.target_spec.supported_ops = [
+        tf.lite.OpsSet.TFLITE_BUILTINS,
+        tf.lite.OpsSet.SELECT_TF_OPS,
+    ]
+
+    converter.allow_custom_ops = True
+    return converter.convert()
+
+
+@click.command()
+@click.option("--num_classes", default=80, help="Number of classes")
+@click.option(
+    "--weights_path", default=None, help="Path to .h5 file with model weights"
+)
+def convert_tflite(num_classes, weights_path):
+    """Creates a .tflite file with a trained YOLOv4 model
+
+    Args:
+        num_classes (int): Number of classes
+        weights_path (str, optional): Path to .h5 pre-trained weights file
+    """
+    model = YOLOv4(
+        input_shape=(HEIGHT, WIDTH, 3),
```

Comment:
Height and width are parametrizable, is it an argument stored in the tflite model or is it just used for the conversion? We want to make sure users can proceed the inference on any size